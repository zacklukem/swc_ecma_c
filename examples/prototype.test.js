function MyConstructor(input) {
  this.a = "Something";
  this.b = "something else";
  this.input = input;
}

MyConstructor.prototype.printMe = function () {
  console.log("a:", this.a);
  console.log("b:", this.b);
  console.log("input:", this.input);
};

let instance1 = new MyConstructor("instance1");
instance1.printMe();

let instance2 = new MyConstructor("instance2");
instance2.printMe();
