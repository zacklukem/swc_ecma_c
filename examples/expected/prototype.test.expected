a: Something
b: something else
input: instance1
a: Something
b: something else
input: instance2
